### ISSUE TITLE
SymPy output exceeds web page width

### ISSUE THREAD

**Issue by agryman** [moorepants/learn-multibody-dynamics#91]
In the SymPy chapter, you evaluate pi to 300 digits. This exceeds the page width. See attached screenshot:

![Screen Shot 2022-06-25 at 3 24 01 PM](https://user-images.githubusercontent.com/8419688/175787923-a49d84dd-63c3-4040-b7f6-83c82b306f14.png)

Is there any way to tell SymPy to fold the output?

**Comment by moorepants** [comment]
Yes, there are a few instances of this. Ideally Mathjax would wrap things appropriately but it doesn't for some statements (see #13). I don't think there are settings in SymPy's latex printer to control wrapping, but if there are/were then we could control these things at the SymPy level also. I think I even opened an issue about this on the jupyter-sphinx repo, as it could be possible to address this at the Jupyter level also. There are differences in wrapping in a Jupyter Notebook session and what is seen in the nbconverted notebook that is rendered here.

**Comment by moorepants** [comment]
Some very informative notes on addressing these is in this SymPy PR: https://github.com/sympy/sympy/pull/26867

**Comment by moorepants** [comment]
On Firefox 147 I see appropriate wrapping (using mathjax v2):

<img width="1649" height="921" alt="Image" src="https://github.com/user-attachments/assets/4cd6fe66-7aab-447f-b0a8-28b16b740bf7" />

in general, but also see some minor overhangs (even thought mathjax is trying to handle the wrapping):

<img width="1598" height="903" alt="Image" src="https://github.com/user-attachments/assets/7e2045bf-59f6-4cac-88ff-f5e36c09587e" />

**Comment by moorepants** [comment]
I have used mathjax v2 because it had wrapping, whereas v3 did not, but it looks like v4 may now have wrapping: https://docs.mathjax.org/en/latest/output/linebreaks.html
